packages/jmix-front-generator/src/building-blocks/stages/writing/pieces/i18n.ts

```diff
@@ -28,7 +29,7 @@ export enum ComponentType {
  * are objects containing i18n key/value pairs for that locale.
  */
 export function writeComponentI18nMessages(
-  fs: MemFsEditor,
+  gen: Generator,
```

Comment:
In other places we are using type `YeomanGenerator`

Reply:
We need it to get the correct destination path.